runtimes/peregrine/src/lib.rs

```diff
@@ -122,16 +123,14 @@ impl frame_system::Config for Runtime {
 	/// The lookup mechanism to get account ID from whatever is passed in
 	/// dispatchers.
 	type Lookup = AccountIdLookup<AccountId, ()>;
-	/// The index type for storing how many extrinsics an account has signed.
-	type Index = Index;
-	/// The index type for blocks.
-	type BlockNumber = BlockNumber;
+	/// The nonce type for storing how many extrinsics an account has signed.
+	type Nonce = runtime_common::Nonce;
```

Comment:
Same here.

Reply:
updated in: [94b4a70](https://github.com/KILTprotocol/kilt-node/pull/559/commits/94b4a70489d1a4fa1503e9729089c2036be4ce4d)